### ISSUE TITLE
Issue merging records with blank attribute matrix attributes

### ISSUE THREAD

**Issue by sichiriu** [SparkDevNetwork/Rock#3995]
### Description
When merging two records with blank attribute matrix attributes, the resulting merged record has a bad value for that attribute.

![Screen Shot 2019-11-13 at 1 46 33 PM](https://user-images.githubusercontent.com/10267139/68808501-b8f3ea80-061e-11ea-8137-2e484ab971c5.png)

### Steps to Reproduce
1. Configure a new person matrix attribute
1. Merge two records that have no value for that attribute
1. Observe a bad value for that attribute in the merged record

**Expected behavior:**

The merge occurs and the merged attribute is blank.

**Actual behavior:**

A bad value is stored in the AttributeValue table.

### Versions
* **Rock Version:** Rock McKinley 9.3 (1.9.3.2)
* **Client Culture Setting:** en-US

**Comment by MrUpsideDown** [comment]
This issue has been fixed in v1.10 with this commit:
https://github.com/SparkDevNetwork/Rock/commit/386627fe6647c423f59b41d8e6b1f83fb2d35109

**Comment by JRichmond27** [comment]
Just to help this come up in searches by other people with this issue...

If you end up with someone with the invalid attribute value (`00000000-0000-0000-0000-000000000000`), and then try to merge that person again, an error is thrown:

> Merge Error
> The following error occurred when attempting the merge: Sequence contains no elements

The only workaround I've found is to run a SQL query to delete the invalid attribute value.

```DELETE FROM AttributeValue WHERE AttributeID = XXXX AND Value = '00000000-0000-0000-0000-000000000000'```

Glad to hear this has been fixed in v10! We'll be updating soon.
